drape/glyph_manager.cpp

```diff
@@ -146,12 +143,14 @@ class Font
 
   bool HasGlyph(strings::UniChar unicodePoint) const { return FT_Get_Char_Index(m_fontFace, unicodePoint) != 0; }
 
-  Glyph GetGlyph(strings::UniChar unicodePoint, uint32_t baseHeight) const
+  Glyph GetGlyph(strings::UniChar unicodePoint, uint32_t glyphHeight) const
   {
-    uint32_t const glyphHeight = baseHeight * m_sdfScale;
+    FREETYPE_CHECK(FT_Set_Pixel_Sizes(m_fontFace, 0, glyphHeight));
 
-    FREETYPE_CHECK(FT_Set_Pixel_Sizes(m_fontFace, glyphHeight, glyphHeight));
+    // FT_LOAD_RENDER with FT_RENDER_MODE_SDF uses bsdf driver that is around 3x times faster
+    // than sdf driver, activated by FT_LOAD_DEFAULT + FT_RENDER_MODE_SDF
     FREETYPE_CHECK(FT_Load_Glyph(m_fontFace, FT_Get_Char_Index(m_fontFace, unicodePoint), FT_LOAD_RENDER));
+    FREETYPE_CHECK(FT_Render_Glyph(m_fontFace->glyph, FT_RENDER_MODE_SDF));
```

Comment:
BTW, we had _delayed_ async glyph rendering logic before. Not to hang the rendering routine.
Now it is rendered in place. Did you visually check how it works? Especially on low-end Androids :)

Reply:
I explicitly mentioned it in the commit and in the PR description. I have tested it on low-end Androids, and didn't see a negative difference (there the most time is taken by slow feature reads and rendering of roads/areas, texts appear almost immediately).

A positive difference is that texts do not blink now as before.

The current performance is comparable to the previous threaded approach, because:
1. There is no threading overhead to pass glyphs in many smaller chunks to 3 different glyph rendering threads.
2. All symbols are now uploaded in a few larger batches by the frontend thread, instead of uploading a few symbols per each rendered frame.
3. Freetype's SDF implementation is faster than the removed one and provides a better visual quality result.

IMO the current state is good enough to release, but it's better to beta-test it first, of course.

The next step will introduce additional HB shaping before rendering glyphs. That step may create a sensible delay in text rendering, in this case, we may need to shape/render different strings on different threads. That is another story, I have some ideas on how to do it cleaner, via a (system) thread pool with callbacks.